src/Tabs/Tabs.js

```diff
@@ -111,6 +111,10 @@ export type Props = {
    */
   textColor?: 'accent' | 'primary' | 'inherit',
   /**
+   * @ignore
+   */
+  theme?: Object,
```

Comment:
@alitaheri I'm adding them back. They are needed for the auto-documentation generation.